Source/Peripheral.swift

```diff
@@ -829,8 +829,7 @@ public class Peripheral {
     /// * `BluetoothError.bluetoothPoweredOff`
     /// * `BluetoothError.bluetoothInUnknownState`
     /// * `BluetoothError.bluetoothResetting`
-    #if os(iOS) || os(tvOS) || os(watchOS)
```

Comment:
Shouldn't it stay? Currently probably it is not needed — technically though it is possible that there will be a new system that will not support the function?

Reply:
It’s following the same pattern as https://github.com/c0diq/RxBluetoothKit/blob/d710c2e105982dc503cb8073c3ff227e95a8b6ae/Source/Peripheral.swift#L418